health-services/facility/LOCALSETUP.md

```diff
@@ -0,0 +1,31 @@
+# Local Setup
+
+To setup the Facility service in your local system, clone the [Health campaign services](https://github.com/egovernments/health-campaign-services).
+
+## Dependencies
+
+### Infra Dependency
+
+- [X] Postgres DB
+- [X] Redis
+- [X] Elasticsearch
+- [X] Kafka
+    - [X] Consumer
+    - [X] Producer
+
+
+You can use docker-compose file to get started with these dependencies. Download docker-compose.yml from [here](../libraries/docker-compose.yml)
+
+Use the following command to start containers
+
+```
+cd path/to/docker-compose.yml file
+
+docker-compose up -d
+```
```

Comment:
Can you move this part in Running Locally section.

Reply:
Resolved